src/RequestModel.php

```diff
@@ -9,6 +9,7 @@
 abstract class RequestModel implements RequestModelInterface
 {
     private array $requestData = [];
+    protected string $attributeDelimiter = '.';
```

Comment:
How about make this nullable and by default set `null` ?

Reply:
It seems to me so more explicitly that a dot will be used